pkg/common/prometheus/metrics.go

```diff
@@ -127,4 +127,22 @@ var (
 		// Possible optype - "create-volume", "delete-volume", "attach-volume", "detach-volume", "expand-volume", etc
 		// Possible status - "pass", "fail"
 		[]string{"optype", "status"})
+
+	// FullSyncCount is a counter metric to count number of completed CSI full sync cycles.
+	FullSyncCount = promauto.NewCounter(prometheus.CounterOpts{
```

Comment:
Histogram metric comes with a counter. Why do you need a separate one?

Reply:
same question. Histogram has count and sum.

Reply:
@lipingxue @SandeepPissay removed the counter metric.